shell/assets/styles/global/_tooltip.scss

```diff
@@ -5,7 +5,7 @@
 
   display: block !important;
   z-index: z-index('tooltip');
-  max-width: 50vw;
+  max-width: 90vw;
```

Comment:
This isn't needed. The issue with long names is due to vtoolip (specifically the popper.js component under the hood) getting confused with it's boundary element and moving the position from right to left. There's another flag which hides the poppover on hover... which then removes the tooltip. This is the weirdness when the cursor changes from disabled to text (text because the popover is covering the cluster icon).

Hacky fix is in getTooltipConfig 

```
      return {
        content:       this.shown ? null : item,
        placement:     'right',
        popperOptions: {
          modifiers: { preventOverflow: { enabled: false } }
        }
      };
```

Proper fix is probably to set `boundariesElement` to the jquery element for body i guess, but lets avoid any jquery mallarky